apps/verticalhome/js/pin_app_navigation.js

```diff
@@ -43,6 +38,9 @@
      */
     reset: function reset() {
       MoreAppsNavigation.clearNavigationEvents();
+      window.removeEventListener('keydown', this);
```

Comment:
I'm just wondering, why do we need to remove event listener before adding it back?